index.html

```diff
@@ -0,0 +1,747 @@
+                
+            <div>
+                    <ul>
+                <li>
+                    <button class="Subscrib-now" type="button">SUBSCRIBE NOW
+                    </button>
+                </li>
+                <li>                    
+                        <button class="Log In" type="button">LOG IN</button>
+                    
+                </li>
+            
+            </ul>
+            </div>
+        </div>
+        </nav>
+    </header>
+     <main>
+        <div class="header">
+            <h1 class="heading">
+                Space Ripples Reveal Big Bang’s Smoking Gun
+            </h1>
+        </div>
+            <div >
+                <figure class="banner">
+                    <img src="assets/images/Alanglut.jpg" alt="Alan Guth was one of the first 
+                    physicists to hypothesize the 
+                    existence of inflation, which explains how the 
+                    universe expanded so uniformly and so quickly in 
+                    the instant after the Big Bang 13.8 billion years ago." width="960" >
+                    <figcaption>
+                        <span>
+                            Alan Guth was one of the first physicists
+                            to hypothesize the existence of inflation, which explains how the
+                            universe expanded so uniformly and so
+                            quickly in the instant after the Big Bang 13.8 billion years ago.
+                        </span>
+                        <span>Rick Friedman for The New York Times</span>
+                    </figcaption>
+
+                </figure>
+        </div>
+        <section>
+            <div class=author-details>
+                <h4 class=author>By Dennis Overbye</h4>
+                <div class="flex-display">
+                <p><time datetime="2014-03-17 20:00">March 17, 2014</time>
+                    <p style="margin-right: 22rem;">
```

Comment:
Please, remove inline styling, it is considered a bad practice, move it to your CSS files.